bs-jest - Cannot find module 'bisect_ppx/lib/js/src/runtime/bucklescript/runtime.js' from 'jest.js'

When we use `bs-jest` and enabled `in-source: true` in `bsconfig.json`, for example:

```json
 "package-specs": [
    {
      "module": "commonjs",
      "in-source": true
    }
  ]
```

we will get an error when running the Jest:

```sh
 FAIL  __tests__/hello_test.js
  ● Test suite failed to run

    Cannot find module 'bisect_ppx/lib/js/src/runtime/bucklescript/runtime.js' from 'jest.js'

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:299:11)

Test Suites: 1 failed, 1 total...
```

it seems like the generated path of `runtimes.js` has been changed because of `in-source` flag:

https://github.com/aantron/bisect_ppx/blob/d0666721e7456abd1f9f8cc211b2813a0b91dc80/src/runtime/bucklescript/jest.js#L1

(`runtime.js` reside in  `bisect_ppx/src/runtime/bucklescript/runtime.js`)


here is reproducible repo:
https://github.com/jihchi/bisect-starter-bsb-jest/tree/cannot_find_module_insource

--- a/README.md
+++ b/README.md
@@ -219,6 +219,8 @@ and install it:
 3. If you are using Jest, add this to your `package.json`:
 
     ```json
+    "setupFilesAfterEnv": ["bisect_ppx/lib/js/src/runtime/bucklescript/jest.js"]
+    // or if you enable "package-specs.in-source" flag in bsconfig.json
     "setupFilesAfterEnv": ["bisect_ppx/src/runtime/bucklescript/jest.js"]
     ```
 

--- a/src/runtime/bucklescript/jest.js
+++ b/src/runtime/bucklescript/jest.js
@@ -1,5 +1,0 @@
-var runtime = require("bisect_ppx/lib/js/src/runtime/bucklescript/runtime.js");
-afterAll(function () {
-  runtime.write_coverage_data();
-  runtime.reset_coverage_data();
-});

--- a/src/runtime/bucklescript/jest.ml
+++ b/src/runtime/bucklescript/jest.ml
@@ -1,0 +1,8 @@
+external afterAll : ((unit -> unit)[@bs ]) -> unit = "afterAll"[@@bs.val ]
+
+let () =
+  afterAll
+    ((fun ()  ->
+        Runtime.write_coverage_data (); 
+        Runtime.reset_coverage_data ())
+    [@bs ])

--- a/src/runtime/bucklescript/jest.ml
+++ b/src/runtime/bucklescript/jest.ml
@@ -4,7 +4,7 @@
       // ... snip ...
     }));
   ```
-  this will cause timeout error in Jest because Jest will wait until you manully call `done` from `param`. *)
+  this will cause timeout error in Jest because Jest will wait until you manually invoke `param.done()` within afterAll. *)
 external afterAll : ((unit -> unit)[@bs ]) -> unit = "afterAll"[@@bs.val ]
 
 let () =

--- a/src/runtime/bucklescript/jest.ml
+++ b/src/runtime/bucklescript/jest.ml
@@ -1,3 +1,10 @@
+(* If we make a curried binding of `afterAll`, BuckleScript will compiles it like this:
+  ```
+    afterAll((function (param) { // <-- notice the `param` parameter here
+      // ... snip ...
+    }));
+  ```
+  this will cause timeout error in Jest because Jest will wait until you manully call `done` from `param`. *)
 external afterAll : ((unit -> unit)[@bs ]) -> unit = "afterAll"[@@bs.val ]
 
 let () =

